### DISCUSSION TITLE
discography/singles/1979-11-09-somewhere-south-of-macon-promo

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#28] [General]
# discography/singles/1979-11-09-somewhere-south-of-macon-promo

This promo single would go on to become the twelveth single released by Lena Zavaroni.

https://fanzoflenazavaroni.github.io/discography/singles/1979-11-09-somewhere-south-of-macon-promo

**Comment by StephenMolloy1** [comment]
I have just come across this on Discogs, it was also released in France some time in 1980. You live and learn.

https://www.discogs.com/release/13040555-Lena-Zavaroni-Somewhere-South-Of-Macon

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the link gave me a reason to start creating individual pages for each version of a single I had already started on the Studio Albums. 

It should help reduce the data load for mobile users and makes an "On This Day" more clear as to which of the versions of a Single/Album the "On This Day" is related to when there are many versions of a Single/Album.

**Comment by StephenMolloy1** [comment]
I wonder if support was withdrawn from this record simply because Lena was in hospital and wouldn't be able to tour around record shops doing signings etc. Maybe it could have been regarded as being in bad taste to promote it.

**Reply by FanzOfLenaZavaroni** [reply]
Good point! It a shame we may never know the reason why, but it would make a nice change if it was for a positive reason that the single's release was delayed.
